web/themes/custom/server_theme/templates/server-theme-container-rounded-corners-big.html.twig

```diff
@@ -1,3 +1,13 @@
-<div class="rounded-lg overflow-hidden w-fit">
+{% import '@server_theme/templates/server-theme-bg-color-base.html.twig' as bgColorBase %}
+
+{% set color_class = bgColorBase.getBgColor(bg_color) %}
+
+{% if color_class|trim != 'light-green' %}
```

Comment:
This seems like a weird special casing. If the twig is generic, why would `light-green` invoke padding?